cache/fast_lru_cache.cc

```diff
@@ -310,14 +295,11 @@ uint8_t LRUCacheShard::CalcHashBits(
   size_t num_entries =
       static_cast<size_t>(capacity / (kLoadFactor * handle_charge));
 
-  // Compute the ceiling of log2(num_entries). If num_entries == 0, return 0.
-  uint8_t num_hash_bits = 0;
-  size_t num_entries_copy = num_entries;
-  while (num_entries_copy >>= 1) {
-    ++num_hash_bits;
+  if (num_entries == 0) {
+    return 0;
   }
-  num_hash_bits += size_t{1} << num_hash_bits < num_entries ? 1 : 0;
-  return num_hash_bits;
+  uint8_t num_hash_bits = static_cast<uint8_t>(FloorLog2<size_t>(num_entries));
```

Comment:
Nit: in accordance with Google C++ style, we use `int` for small counts such as number of bits. (I use `uint8_t` if it's a value that might be used in either a unsigned (e.g. `uint32_t`) or signed (e.g. `int`) context.)